pkg/types/hashedrekord/v0.0.1/entry_test.go

```diff
@@ -284,6 +284,13 @@ func TestCrossFieldValidation(t *testing.T) {
 			t.Errorf("unexpected result in '%v': %v", tc.caseDesc, err)
```

Comment:
Do we want to test a failure case for Insertable? Maybe at least for the primary three types, hashedrekord, rekord, intoto?

Reply:
Probably a good idea